RLTest/env.py

```diff
@@ -28,15 +28,19 @@ def method(*argc, **nargs):
 
 
 class Query:
-    def __init__(self, env, *query):
+    def __init__(self, env, *query, **kwargs):
```

Comment:
Can you explain why you needed this feature?

Reply:
To allow something like: `env.expect(..., conn: r).equal(...)` when `r` is a cluster connection.

Reply:
To allow `env.expect(..., conn: r).equal(...)` with `r` as a cluster connection.